docs/site/nbb.md

```diff
@@ -0,0 +1,13 @@
+# Using Calva with nbb
+
+Since [nbb](https://github.com/borkdude/nbb) can be started such that it is an nREPL server, Calva can connect to it and a lot of the features will work.
+
+Calva can also start nbb and connect its REPL for your, using the Jack-in command. This will start an nbb nREPL server on a random port and connect Calva to it.
```

Comment:
```suggestion
Calva can also start nbb and connect its REPL for you, using the Jack-in command. This will start an nbb nREPL server on a random port and connect Calva to it.
```